### PR TITLE
fixed SHOWING-'This page isn’t working'

### PR DESCRIPTION
**Pull request by shubhankarsharma00**
with respect to issue #141

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -345,4 +345,4 @@ DEPENDENCIES
   will_paginate-bootstrap
 
 BUNDLED WITH
-   1.17.2
+   1.17.3
```
Comment (ankitkataria) [inline Gemfile.lock]: Should there be a particular version of `bundler` to be used mentioned in the readme? Like we have versions mentioned for `rails`, `ruby` and `postgres`. Changing the bundler version in the gemfile might create problems for other users.

Comment (tachyons) [reply Gemfile.lock]: Bundler versions hardly cause any issues.

Comment (tachyons) [reply Gemfile.lock]: Except when it is major release. Like bundler 1.x and bundler 2.x

**Comment by shubhankarsharma00** [comment]
Is the PR okay? Why isn't it approved @tachyons is there something wrong?

**Pull request merged**